doc/releases/changelog-dev.md

```diff
@@ -16,6 +16,45 @@
 
 <h3>Improvements</h3>
 
+* The QNode has been re-written to support batch execution across the board,
+  custom gradients, better decomposition strategies, and higher-order derivatives.
+  [(#1807)](https://github.com/PennyLaneAI/pennylane/pull/1807)
+
+  - Internally, if multiple circuits are generated for execution simultaneously, they
```

Comment:
```suggestion
  - Internally, if multiple circuits are generated for simultaneous execution, they
```
Maybe I'm getting wrong here what you want to say, but the time of generation is less important than the execution time (which is what should happen in parallel), right? :thinking: